### PR TITLE
rpcserver: add handleTxFeeInfo tests.

### PR DESCRIPTION
**Pull request by dnldd**
This adds rpc tests for `handleTxFeeInfo`.

Work towards #2069.

### CODE REVIEW

**Comment by dnldd** [comment]
```sh
go tool cover -func=cov.out | grep -E handleTxFeeInfo
github.com/decred/dcrd/internal/rpcserver/rpcserver.go:5054:              handleTxFeeInfo                 100.0%
```

File: internal/rpcserver/rpcserverhandlers_test.go
Comment (rstaudt2) [inline internal/rpcserver/rpcserverhandlers_test.go:5994]: Same nit as in the other PR to use `errors.new` here instead for consistency with the other tests, and since no formatting is required.

**Review by rstaudt2 — CHANGES_REQUESTED** [review]

File: internal/rpcserver/rpcserverhandlers_test.go
Comment (rstaudt2) [reply internal/rpcserver/rpcserverhandlers_test.go:159]: This comment can be marked resolved - saw that this was not needed for this PR, but was needed in https://github.com/decred/dcrd/pull/2408 that you have on top of this to handle mocking when the function is called multiple times within a test

**Review by davecgh — APPROVED** [review]

**Comment by davecgh** [comment]
Needs rebase, but probably should wait until #2406 is rebased and merged since it'll need another one after that.

**Pull request merged**
